apps/aefate/src/aefa_utils.erl

```diff
@@ -133,4 +135,27 @@ decode_variant([], [], _N, _Values, '$undefined$') ->
 decode_variant([], [], _N, _Values, Acc) ->
     Acc.
 
-
```

Comment:
This is an internal test format, it should not be moved from `aefate/test` to `aefate/src`

Reply:
done